torch/_dynamo/allowed_functions.py

```diff
@@ -152,6 +152,7 @@ def _is_allowed_module_prefix(obj):
         # these functions, rather than keep them opaque-ly in the graph.
         disallowed_modules = (
```

Comment:
Add "torch.ao.nn" to this disallowed_modules can fix the internal failures. Because builtin modules under ```torch.ao.nn```:
* Don't pass the previous check: ```module_attr.startswith("torch.nn.")```;
* Pass the new check: ```is_allowed(module_where_fn_is_defined)```.

We don't treat quantized modules as allowed, though I'm not 100% sure if this is expected.